app/src/main/res/layout/main_fragment.xml

```diff
@@ -26,4 +26,13 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="0.498"
         app:layout_constraintStart_toStartOf="parent" />
+
+    <Button
+        android:id="@+id/button2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Create WatchList"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        tools:layout_editor_absoluteY="164dp" />
```

Comment:
This is more of a UI change, when the person logs into the app, have the ability to create their watchlist.

Reply:
Ditto comments above.  This is incomplete functionality.
1) Only features that meet the Definition of Done should be merged to Master.
2) Unused code is clutter, which increases technical debt.